app/views/posts/show.html.erb

```diff
@@ -23,4 +23,5 @@
         </li>
       <% end %>
     </ul>
-  <% end %>
\ No newline at end of file
+  <% end %>
+  
```

Comment:
- [ ] [OPTIONAL AND IMPORTANT] For a good UX there should be a button to add comment and also add likes in you post-show page, currently I have to manually change routes to achieve this, consider adding those buttons